From 74f0ef93a4fe37165c691bc20a90d83cf3b8e5db Mon Sep 17 00:00:00 2001
Message-ID: <74f0ef93a4fe37165c691bc20a90d83cf3b8e5db.1726926634.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1726926633.git.sharlatanus@gmail.com>
References: <cover.1726926633.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 18 Sep 2024 13:15:12 +0100
Subject: [PATCH v2 06/25] gnu: python-ndcube: Disable failing tests.

* gnu/packages/astronomy.scm (python-ndcube) [arguments]: <#test-flags>:
Disable 2 tests failing after python-astropy updated to 6.1.3.

Change-Id: I95d07fb5d26dda534afbbe44d5dd9f30ebbf6d09
---
 gnu/packages/astronomy.scm | 13 ++++++++++---
 1 file changed, 10 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index acfc603153..8cb25aab08 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3061,9 +3061,16 @@ (define-public python-ndcube
     (build-system pyproject-build-system)
     (arguments
      (list
-      ;; Break cycle: python-ndcube -> python-specutils -> python-ndcube, see
-      ;; <https://github.com/sunpy/ndcube/issues/733>.
-      #:test-flags #~(list "-k" "not test_rebin_specutils")
+      #:test-flags
+      #~(list "-k" (string-append
+                    ;; Break cycle: python-ndcube -> python-specutils ->
+                    ;; python-ndcube, see
+                    ;; <https://github.com/sunpy/ndcube/issues/733>.
+                    "not test_rebin_specutils"
+                    ;; Introduced with astropy 6.1.3, see
+                    ;; https://github.com/sunpy/ndcube/issues/758>.
+                    " and not test_2d[celestial_2d_ape14_wcs]"
+                    " and not test_2d[celestial_2d_fitswcs]"))
       #:phases
       #~(modify-phases %standard-phases
           (add-after 'unpack 'break-cycle
-- 
2.46.0

